packages/wrangler/src/deployment-bundle/run-custom-build.ts

```diff
@@ -7,6 +7,50 @@ import { execaCommand } from "execa";
 import { logger } from "../logger";
 import type { Config } from "@cloudflare/workers-utils";
 
+export async function runCommand(
+	command: string,
+	cwd: string | undefined,
+	prefix = "[custom build]"
+) {
+	logger.log(chalk.blue(prefix), "Running:", command);
+	try {
+		const res = execaCommand(command, {
+			shell: true,
+			...(cwd && { cwd: cwd }),
```

Comment:
can't you just pass `cwd` even if it is `undefined`? 🤔 
```suggestion
			cwd,
```